orderer/common/follower/follower_chain.go

```diff
@@ -82,34 +100,39 @@ type Chain struct {
 	chainCreationCallback CreateChain       // A method that creates a new consensus.Chain for this channel, to replace the current follower.Chain
 
 	cryptoProvider bccsp.BCCSP
-	amIInChannel   cluster.SelfMembershipPredicate
 }
 
 // NewChain constructs a follower.Chain object.
 func NewChain(
-	support Support,
+	resources Resources,
 	joinBlock *common.Block,
 	options Options,
-	createBlockPullerFunc CreateBlockPuller, // A method that creates a block puller on demand
+	createBlockPullerFunc CreateBlockPuller,
 	chainCreationCallback func(),
 	cryptoProvider bccsp.BCCSP,
-	amIInChannel cluster.SelfMembershipPredicate,
 ) (*Chain, error) {
+
+	// Check it once
```

Comment:
nit: what is "it" here? If we're going to have a comment, I think it should be more helpful for someone coming into the code without any knowledge of this area

Reply:
Done